test/resources/beta/referral.test.js

```diff
@@ -43,6 +43,7 @@ describe('Referral Resource', function () {
     const referralsArray = referrals.referral_customers;
```

Comment:
It's unfortunate that this didn't fail, I guess this turned into an empty list of 0 so the assertion itself passed...